lib/internal/modules/esm/get_format.js

```diff
@@ -60,7 +60,7 @@ function getFileProtocolModuleFormat(url, context, ignoreErrors) {
       const config = getPackageScopeConfig(url);
       const fileBasename = basename(filepath);
       const relativePath = StringPrototypeSlice(relative(config.pjsonPath, filepath), 1);
-      suggestion = 'Extension-less files are unsupported inside of ' +
+      suggestion = 'Loading extension-less files is not supported inside of ' +
```

Comment:
```suggestion
      suggestion = 'Loading extensionless files is not supported inside of ' +
```

Reply:
will make an update to the .md so they can cross ref at least then I think we are good to go

Reply:
@ljharb see https://github.com/nodejs/node/pull/42301/commits/36f25aa78eb269b5729cb2ca083a977061f7fd75

Reply:
oh thanks! "view changes" didn't show it :-)